src/context/RegionContext.js

```diff
@@ -95,7 +95,7 @@ export function RegionProvider(props) {
     });
     setRegions(regions);
     setRegionCount(count);
-
+    console.log(regions);
```

Comment:
If we need to leave logging in, use the `loglevel` utility (search the code base for existing uses).

Reply:
@nmcharlton sorry, missed this console log. Fixed it!